packages/storybook/config/manager-head.html

```diff
@@ -70,56 +73,71 @@
     transition: 0.15s ease-in-out;
   }
   
+  .os-content .sidebar-item button[data-selected="true"] {
+    background-color: rgb(var(--inovex-elements-n-3)) !important;
```

Comment:
Overriding styles with !important is always risky. It makes it difficult to override this style at a later point. Is there really no way to reach higher rule specifity with css selectors / import ordering?